lms/djangoapps/ccx/views.py

```diff
@@ -247,7 +247,9 @@ def create_ccx(request, course, ccx=None):
     )
 
     assign_staff_role_to_ccx(ccx_id, request.user, course.id)
-    add_master_course_staff_to_ccx(course, ccx_id, ccx.display_name)
+    is_course_licensing_enable = configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
```

Comment:
Please, add a blank line after this line to separate assignment lines from action lines. @Jacatove